docs/src/lib/functions.md

```diff
@@ -0,0 +1,86 @@
+```@meta
+CurrentModule = DataFrames
+```
+
+# Functions
+
+```@index
+Pages = ["functions.md"]
+```
+
+## Joins
+
+```@docs
+join
+```
+
+## Reshaping
+
+```@docs
+melt
+stack
+unstack
+stackdf
+meltdf
+```
+
+## Data Manipulation
+
+```@docs
+aggregate
+by
+colwise
+groupby
+```
+
+## DataFrame Manipulation
+
+```@docs
+categorical!
+combine
+deleterows!
+dropnull
+dropnull!
+names!
+nullable!
+rename!
+rename
+unique!
+```
+
+## Iteration
+
+```@docs
+eachcol
+eachrow
+```
+
+## Descriptive
+
+```@docs
+completecases
+describe
+eltypes
+names
+nonunique
+ncol
+nrow
+showcols
+size
+```
+
+## IO
+
+```@docs
+head
```

Comment:
`head` and `tail` are not I/O. `printtable` should probably not be exported. `show` should be with `showcols`.